package.json

```diff
@@ -25,32 +25,31 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "bower": "^1.8.2",
     "ember-cli-babel": "^6.3.0",
     "ember-cli-htmlbars": "^2.0.1",
-    "ember-runtime-enumerable-includes-polyfill": "^1.0.1"
+    "ember-runtime-enumerable-includes-polyfill": "^2.0.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "3.0.0",
-    "ember-cli": "^2.14.1",
+    "ember-ajax": "^3.0.0",
```

Comment:
i think we could actually remove ember-ajax entirely here? it's part of the default blueprint but I don't think we are using it are we?

Reply:
Nope, good call.